src/main/java/Deadline.java

```diff
@@ -0,0 +1,43 @@
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Objects;
+
+public class Deadline extends Task {
+
+    protected Date by;
+
+    /**
+     * Constructor for Deadline.
+     *
+     * @param description title of Deadline
+     * @param by datetime for deadline
+     */
+    public Deadline(String description, Date by) {
+        super(description);
+        this.by = by;
+    }
+
+    @Override
+    public String toString() {
+        SimpleDateFormat formatter = new SimpleDateFormat("dd/MM/yyyy HH:mm");
```

Comment:
I think the formatter should be a static variable :) since it is always the same format